src/plugins/charts/public/services/colors/colors.test.ts

```diff
@@ -138,7 +138,7 @@ describe('Vislib Color Service', () => {
   });
 
   it('should return the first hex color in the seed colors array', () => {
```

Comment:
Is this description still right? Does it return hex colors?

Reply:
Yes it does and the test pass. `euiPaletteColorBlind()[0]` is `#54B399`